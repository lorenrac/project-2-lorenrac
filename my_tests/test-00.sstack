// Tons of scopes
PRINT "Scope 0"
{
  PRINT "Scope 1"
  {
    PRINT "Scope 2"
  }
  PRINT "Scope 1 again"
  {
    PRINT "Scope 2 again"
    {
      PRINT "Scope 3"
      {
        PRINT "Scope 4"
      }
      PRINT "Scope 3 again"
      {
        PRINT "Scope 4 again"
        {
          PRINT "Scope 5"
        }
        PRINT "Scope 4 again"
      }
      PRINT "Scope 3 again"
    }
    PRINT "Scope 2 again"
  }
  PRINT "Scope 1 again"
}
PRINT "Scope 0 again"
