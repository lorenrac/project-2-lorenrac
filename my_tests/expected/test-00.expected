Scope 0
Scope 1
Scope 2
Scope 1 again
Scope 2 again
Scope 3
Scope 4
Scope 3 again
Scope 4 again
Scope 5
Scope 4 again
Scope 3 again
Scope 2 again
Scope 1 again
Scope 0 again
